torchvision/datasets/sstem.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import print_function
+
+import os
+
+from PIL import Image
+import skimage
+import torch.utils.data as data
+
+from .utils import download_url, check_integrity
+
+
+class ssTEM(data.Dataset):
+    """Dataset for `ISBI Challenge: Segmentation of neuronal structures
+    in EM stacks <http://brainiac2.mit.edu/isbi_challenge/>`_.
+
+    Args:
+        root (string): Root directory where dataset exists or will be saved
+            to if download is set to True.
+        train (bool, optional): If True, creates dataset from training set,
+            otherwise creates from test set.
+        transform (callable, optional): A function/transform that takes in a
+            PIL image and returns a transformed version.
+        target_transform (callable, optional): A function/transform that takes
+            in the target and transforms it.
+        download (bool, optional): If true, downloads the dataset from the
+            internet and puts it in root directory. If dataset is already
+            downloaded, it is not downloaded again.
+    """
+    base_url = 'http://brainiac2.mit.edu/isbi_challenge/sites/default/files/'
+
+    meta = {
+        'train': {
+            'data': {
+                'filename': 'train-volume.tif',
+                'md5': '465461edbe0254630c4ec5577f1e7764'
+            },
+            'labels': {
+                'filename': 'train-labels.tif',
+                'md5': '657fe6b728c6dd0152e295c6d800001d'
+            }
+        },
+        'test': {
+            'data': {
+                'filename': 'test-volume.tif',
+                'md5': '9767660d7abe4e0ecbdd0061a16058ad'
+            }
+        }
+    }
+
+    def __init__(self, root, train=True, transform=None, target_transform=None,
+                 download=False):
+        self.root = os.path.expanduser(root)
+        self.train = train
+        self.transform = transform
+        self.target_transform = target_transform
```

Comment:
The separate handling of `transform` and `target_transform` is only for backward compatibility. You should use `transforms` now. See #872.